zellij-tile/src/lib.rs

```diff
@@ -1,3 +1,4 @@
+pub mod actions;
```

Comment:
I'd personally merge this file in with the `data` module, but if you disagree, I'm happy to chat about it!

Reply:
Merged into data https://github.com/zellij-org/zellij/pull/319/commits/9c0767af44e15b56946b555a2044c22fe41cc447